config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -0,0 +1,255 @@
+presubmits:
+  kubernetes/dns:
+  - name: pull-kubernetes-dns-test
+    agent: kubernetes
+    context: pull-kubernetes-dns-test
+    branches:
+    - master
+    rerun_command: "/test pull-kubernetes-dns-test"
+    trigger: "(?m)^/test( all| pull-kubernetes-dns-test),?(\\s+|$)"
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --root=/go/src
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --scenario=execute
+        - --
+        - "true"
```

Comment:
whoever made this job probably should run some real test ?!

Reply:
Humm, I thought we run real test.